plugin/trino-hive/src/main/java/io/trino/plugin/hive/ViewReaderUtil.java

```diff
@@ -201,14 +202,14 @@ public ConnectorViewDefinition decodeViewData(String viewSql, Table table, Catal
                         columns,
                         Optional.ofNullable(table.getParameters().get(TABLE_COMMENT)),
                         Optional.empty(),
-                        false);
+                        runHiveViewsAsInvoker);
             }
             catch (RuntimeException e) {
                 throw new TrinoException(HIVE_VIEW_TRANSLATION_ERROR,
                         format("Failed to translate Hive view '%s': %s",
                                 table.getSchemaTableName(),
                                 e.getMessage()),
-                        e);
+                                e);
```

Comment:
This formatting change doesn't seem right

Reply:
Misread this; thought I was fixing the indentation but the original was correct.